guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc

```diff
@@ -41,10 +41,6 @@ ifdef::satellite[]
 +
 * If your deployment uses `katello-agent` and *goferd*, enter `katello-agent`.
 * If your deployment does not use `katello-agent` and *goferd*, enter `katello-host-tools`.
-+
```

Comment:
If your deployment does not use `katello-agent` and *goferd*, you are given two options in 6.12:
- *via remote execution*
- *via remote execution - customize first*

Removing reference to closed WONTFIX BZ is OK but you are also removing reference to `via remote execution` w/o any replacement

Reply:
Update:  on 6.12 there can still be **via Katello Agent** if KA is enabled 

so it is the same for both 6.11 and 6.12